src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -32,25 +40,45 @@ public void switchToScoringPipeline() {
   }
 
   public void periodic() {
-    Pose2d pose = getPureLimelightRobotPose();
-    if (pose != null) {
-      SmartDashboard.putNumber("PoseX", pose.getX());
-      SmartDashboard.putNumber("PoseY", pose.getY());
-      SmartDashboard.putNumber("Rotation", pose.getRotation().getDegrees());
+    Pose2d limelightOnePose = getPureLimelightRobotPose();
+    Pose2d limelightTwoPose = getPureSecondLimelightRobotPose();
+    if (limelightOnePose != null) {
+      SmartDashboard.putNumber("PoseX", limelightOnePose.getX());
+      SmartDashboard.putNumber("PoseY", limelightOnePose.getY());
+      SmartDashboard.putNumber("Rotation", limelightOnePose.getRotation().getDegrees());
+    } else {
+      SmartDashboard.putNumber("PoseX", 0);
+      SmartDashboard.putNumber("PoseY", 0);
+      SmartDashboard.putNumber("Rotation", 0);
+    }
+
+    if (limelightTwoPose != null) {
+      SmartDashboard.putNumber("PoseX", limelightTwoPose.getX());
```

Comment:
might want to change the name of the SmartDashboard entries to be different per limelight.